.ci/settings/settings.py

```diff
@@ -1,2 +1,3 @@
 CONTENT_ORIGIN = "http://localhost:8080/"
 ALLOWED_EXPORT_PATHS = ["/tmp"]
+ORPHAN_PROTECTION_TIME = 0
```

Comment:
@mdellweg seems like some rpm content was not being cleaned up in between tests. This seems to have fixed it.